src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands.ServiceFabric.Netcore.csproj

```diff
@@ -48,6 +43,10 @@
     <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Common.Authentication.Abstractions.Netcore.csproj" />
     <ProjectReference Include="..\..\..\Common\Commands.Common.Graph.RBAC\Common.Rbac.Netcore.csproj" />
     <ProjectReference Include="..\..\..\Common\Commands.Common\Common.Netcore.csproj" />
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Keyvault\Common.Keyvault.Netcore.csproj" />
```

Comment:
@praries880 I believe the build is failing because of capitalization -- Key**v**ault vs. Key**V**ault ☺️